Check topic id if chat is a topic

enyineer opened the pull request:
Fixes my request for fix comment in #14

spkesDE commented:
Looks good, but I would change it so it a own card, so users don't get confused. if you want to do it feel free, or I'll do it once I find some time.

enyineer commented:
Sure, I'll give it a try tomorrow after work 😃 
Happy to help.

enyineer commented:
@spkesDE 

Things I've improved:

1. Removed topics from Chat Trigger Autocompletion
![pr-2](https://github.com/spkesDE/com.spkes.telegramNotifications/assets/27201677/222a0ee0-31fe-45e8-8891-012b697049dd)
2. Added a Trigger Card specifically for Topics
![pr-3](https://github.com/spkesDE/com.spkes.telegramNotifications/assets/27201677/3408d52a-2d5d-4a5f-a2a5-c9d0dced8542)
3. I did also change multiple args titles / translations from "User" to "Chat / Topic" as they can be either a private Chat, Group Chat or Topic
4. Action Cards still include both Chats and topics as it would probably become too confusing having duplicated Actions for Chats and Topics (just the quantity of Cards that would result by splitting them up).

spkesDE commented on flow/triggers/receiveMessageFromChat.ts:
```diff
@@ -8,27 +8,18 @@ export default class ReceiveMessageFromChat {
         card.registerArgumentAutocompleteListener(
             'chat',
-            async (query) => Utils.userAutocomplete(app.chats, query)
+            async (query) => Utils.userAutocomplete(app.chats, query, { skipTopics: true })
```
I like the use of an object for settings

spkesDE commented:
https://homey.app/de-de/app/com.spkes.telegramNotifications/Telegram/test/

 Deine Änderungen sind nun im Test Branch.
 
Erstmal danke für die Umsetzung und Hilfsbereitschaft. Die Änderungen sehen für mich als "hobby" Entwickler gut aus. 
Ich hab dich in den Mitwirkende hinzugefügt. Ich hoffe das es okay für dich.

Falls du mehr Ideen hast oder was besprechen willst kannst du mir gerne auf Telegram schreiben. https://t.me/julian022

enyineer commented:
Alles klar, danke dir 🙂
Vielleicht finde ich bei intensiver Nutzung der App noch andere Dinge die wir zukünftig optimieren können / wollen. Ich helfe da auf jeden Fall gerne aus wenn du etwas hast wo ich aushelfen kann.
